packages/babel-make-styles/README.md

```diff
@@ -92,3 +92,31 @@ const useStyles = __styles({
   },
 });
 ```
+
+## Troubleshooting
+
+This section focuses mainly on troubleshooting this babel plugin in the [microsoft/fluenui repo](https://github.com/microsoft/fluentui).
+However the concepts are not coupled to the repo setup.
+
+### Linaria
+
+The plugin uses tools from [linaria](https://github.com/callstack/linaria) to evaluate runtime calls of `makeStyles`.
+[Linaria's debugging documentation can help here](https://github.com/callstack/linaria/blob/master/CONTRIBUTING.md#debugging-and-deep-dive-into-babel-plugin). Here are a few examples with building packages with linaria debug output.
+
+Directly from the package
+
+> DEBUG=linaria\* LINARIA_LOG=debug yarn build
+
+Using Lage from the root of the repo
+
+> DEBUG=linaria\* LINARIA_LOG=debug yarn lage build --to <package-name>
+
+Using yarn workspace from the root of the repo
+
+> DEBUG=linaria\* LINARIA_LOG=debug yarn workspace <package-name> build
+
+The debug output will include:
+
+- Transformed code
+- Evaluated code
+- AST that indicates what code has been shaken with `@linaria/shaker`
```

Comment:
````suggestion
- AST that indicates what code has been shaken with `@linaria/shaker`

```
linaria:evaluator:shaker:shake  source (exports: --- 💡 exports imported from this file ---):
--- 💡 transformed code that will be used by @linaria/shaker ---
linaria:evaluator:shaker:shake  --- 💡 printed AST with results of shaking ---	
linaria:evaluator:shaker:generate  Generate shaken source code filename.js
linaria:sub-module-1:evaluate  filename.js (only default): --- 💡 code after shaking ---
```
````

not sure that it's useful, but I suggest to describe log output a bit

Reply:
I dunno how useful to have this, since it's possible that Linaria will change the output format at some point. I opted just for general info about the output

Reply:
Agree, let's ignore this comment then 😜